pilot/test/util/diff.go

```diff
@@ -17,18 +17,18 @@ package util
 import (
 	"errors"
 	"io/ioutil"
-	"os"
 	"strings"
 	"testing"
 
 	"github.com/pmezard/go-difflib/difflib"
+
+	"istio.io/istio/pkg/env"
 )
 
 // Refresh controls whether to update the golden artifacts instead.
 // It is set using the environment variable REFRESH_GOLDEN.
 func Refresh() bool {
-	v, exists := os.LookupEnv("REFRESH_GOLDEN")
-	return exists && v == "true"
+	return env.RegisterBoolVar("REFRESH_GOLDEN", false, "").Get()
```

Comment:
This is for public documentation right? I don't think we want to document variables used in tests only?

Reply:
Yes, pkg/env is for documentation. The environment variables end up being documented per published binary, so there will be no doc produced for test binaries. I just added this to shut up the linter.

Reply:
I think we should make the linter not lint test files. It's unnecessary to document test environment variables.